src/react-vite/03-intro-to-jsx/intro-to-jsx.md

```diff
@@ -2,33 +2,46 @@
 @parent learn-react-vite 3
 @outline 3
 
-@description TODO
+@description Learn how to use JSX to define your UI in React.
 
 @body
 
 ## Overview
 
-TODO
+In this section we will:
 
-## Objective 1
+- Learn to use JSX to define our UI
+- Install project assets
+- Create our first component
+- Add variables to JSX
+- Work with loops in JSX
 
-TODO
+## Objective 1 - Creating a UI with JSX
```

Comment:
Let’s format these as `Objective 1: Creating a UI with JSX` (to be consistent with the others).